src/Domain/Enums/Currencies.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace DnDCharacterSheet.Domain.Enums;
+public enum Currencies
+{
+    CopperPieces,
```

Comment:
Extract PIeces and concat

Reply:
I don't think this is necessary because calling it CopperPieces, CopperCoins or just Copper doesn't really matter. It is true that "Pieces" does not provide relevant data but if we want to refactor this we can use powerful IDEs like Visual Studio Code 2022 (Ctrl + R Ctrl + R goes brrr)